src/main/docker/Dockerfile

```diff
@@ -27,20 +27,26 @@ ENV TZ=Etc/UTC \
     PATH="/opt/java/openjdk/bin:${PATH}" \
     LANG=C.UTF-8
 
-COPY --from=jre-build /opt/java/openjdk $JAVA_HOME
-
-# Copy the compiled WAR to the application directory created above
-# Automatically creates the $APP_DIR directory
-COPY ./target/${WAR_FILENAME} ${APP_DIR}
-
-# Create the directory where Dependency-Track will store its data (${DATA_DIR}) and the external library directory (${EXTLIB_DIR})
+# Create the directories where the WAR will be deployed to (${APP_DIR}) and Dependency-Track will store its data (${DATA_DIR})
 # Create a user and assign home directory to a ${DATA_DIR}
 # Ensure UID 1000 & GID 1000 own all the needed directories
-RUN mkdir -p -m 770 ${DATA_DIR} \
+RUN mkdir -p -m 770 ${APP_DIR} \
+    && mkdir -p -m 770 ${DATA_DIR} \
```

Comment:
```suggestion
RUN mkdir -p -m 770 ${APP_DIR} ${DATA_DIR} \
```

Reply:
`g=u` only says "same permissions for group as for user", but it doesn't specify `rwx`. Without `-m 770`, the directory permissions are `775`.

Reply:
true, but I am wondering if we should care if `other` has read and execute rights for the folders ;)

Reply:
Probably not. When using default volumes, `other` can't access the directory on the host system anyway. And when mounted as host volume, the permissions of that host volume apply. That's for `DATA_DIR` which contains key pairs, for `APP_DIR` it doesn't matter at all. I'm going to remove the `-m` flags.